source/com.microsoft.tfs.client.clc/src/com/microsoft/tfs/client/clc/vc/commands/CommandResolve.java

```diff
@@ -203,7 +203,9 @@ public void run() throws ArgumentException, MalformedURLException, CLCException,
          */
         boolean containsSymlink = false;
         for (final Conflict conflict : conflicts) {
-            if (FileSystemUtils.getInstance().getAttributes(conflict.getTargetLocalItem()).isSymbolicLink()) {
+            if (conflict.getTargetLocalItem() != null
+                && conflict.getTargetLocalItem().length() > 0
```

Comment:
Doesn't the third option || resolution.equals(Resolution.OVERWRITE_LOCAL looks strange here? The message we display says: "Symbolic link conflicts can only be resolved by keeping the local version (KeepYours) or taking the server version 
 (TakeTheirs)."